docs/index.html

```diff
@@ -1,443 +1,438 @@
+            event.preventDefault();
+            panel.classList.toggle( 'collapsed' );
 
-			console.log([
-				'    __     __',
-				' __/ __\\  / __\\__   ____   _____   _____',
-				'/ __/  /\\/ /  /___\\/ ____\\/ _____\\/ _____\\',
-				'\\/_   __/ /   _   / /  __/ / __  / / __  /_   __   _____',
-				'/ /  / / /  / /  / /  / / /  ___/ /  ___/\\ _\\/ __\\/ _____\\',
-				'\\/__/  \\/__/\\/__/\\/__/  \\/_____/\\/_____/\\/__/ /  / /  ___/',
-				'                                         / __/  /  \\__  \\',
-				'                                         \\/____/\\/_____/'
-			].join('\n'));
+        };
 
-			filterInput.value = extractQuery();
-			updateFilter( )
 
-		</script>
-		<script src="../build/three.min.js"></script> <!-- console sandbox -->
-	</body>
+        // Functionality for search/filter input field
+
+        filterInput.oninput = function ( event ) {
+
+            updateFilter();
+
+        };
+
+
+        // Functionality for filter clear button
+
+        clearFilterButton.onclick = function ( event ) {
+
+            event.preventDefault();
+
+            filterInput.value = '';
+            updateFilter();
+
+        };
+
+
+        // Activate content and title change on browser navigation
+
+        window.onpopstate = createNewIframe;
+
+
+        // Create the navigation panel and configure the iframe
+
+        createNavigation();
+        createNewIframe();
+
+
+// ----------------------------------------------------------------------------
+// Navigation Panel
+// ----------------------------------------------------------------------------
+
+
+        function createNavigation () {
+
+            // Create the navigation panel using data from list.js
+
+            var navigation = createAndAppendDOMElement( { type: 'div', parent: panel } );
+
+            for ( var section in list ) {
+
+                // Create sections
+
+                var categories = list[ section ];
+
+                var sectionHead = createAndAppendDOMElement( { type: 'h2', parent: navigation, content: section } )
+
+                for ( var category in categories ) {
+
+                    // Create categories
+
+                    var pages = categories[ category ];
+
+                    var categoryContainer = createAndAppendDOMElement( { type: 'div', parent: navigation } );
+                    var categoryHead = createAndAppendDOMElement( { type: 'h3', parent: categoryContainer, content: category } );
+                    var categoryContent = createAndAppendDOMElement( { type: 'ul', parent: categoryContainer } );
+
+                    for ( var pageName in pages ) {
+
+                        // Create page links
+
+                        var pageURL = pages[ pageName ];
+
+                        var listElement = createAndAppendDOMElement( { type: 'li', parent: categoryContent } );
+                        var linkElement = createAndAppendDOMElement( { type: 'span', parent: listElement, content: pageName } );
```

Comment:
@jaxry Thank you for insisting ;) - you are completely right. Seems that I was too much influenced by the current version of the docs where this doesn’t work correctly.

Had not much time this evening, and now it’s too late. Will make a new PR with your improvement and with a "non-hardcoded" redirection tomorrow (similar to what @mrdoob suggested; this is working, too).